File: Herencia/Ejercicio3/Python/Estudiante.py

from Persona import Persona
class Estudiante(Persona):
    def __init__(self, nombre, apellido, ci, celular, d, m, a, ru, sem, fi):
        super().__init__(nombre, apellido, ci, celular, d, m, a)
        self.ru = ru
        self.semestre = sem
        self.fecha_Ingreso = fi
    
    def mostrar(self):
        super().mostrar()
        print(f"RU: {self.ru}")
        print(f"Semestre: {self.semestre}")
        print(f"Fecha de ingreso: {self.fecha_Ingreso}")

File: Herencia/Ejercicio3/Java/Estudiante.java
package Herencia.Ejercicio3.Java;


public class Estudiante extends Persona {
    private int ru, semestre;
    String fecha_Ingreso;
    

    public Estudiante (String nom, String ap, int ci, int cel, int d, int m, int a, int ru, int sem, String fechaIng) {
        super(nom, ap, ci, cel, d, m, a);
        this.ru= ru;
        this.semestre = sem;
        this.fecha_Ingreso = fechaIng;
    }

    void mostrar() {
        super.mostrar();
        System.out.println("RU: " + ru);
        System.out.println("Semestre: " + semestre);
        System.out.println("Fecha de ingreso: " + fecha_Ingreso);
    }
}
